neural_compressor/adaptor/torch_utils/weight_only.py

```diff
@@ -86,7 +86,7 @@ def qdq_weight_sym(weight, num_bits=4, quantile=1.0, return_int=False, full_rang
         minq = torch.tensor(2 ** (num_bits - 1) - 1)
     max_val = torch.max(weight, 1)[0]
     min_val = torch.min(weight, 1)[0]
-    flip_flag = torch.abs(min_val) > torch.abs(max_val)
+    flip_flag = torch.abs(max_val) > torch.abs(min_val)
     wmax = torch.max(torch.abs(max_val), torch.abs(min_val))
```

Comment:
it's a little difficult to understand. why not just using code like this mapping_max = torch.abs(max_value)>torch.abs(min_value)? max_value: min_value, scale = mapping_max/-8. Maybe there are some bugs in my code ~

Reply:
You're right! I also want to simplify this code, your proposal is promising, and I will have a try later.